extensions/metrics/README.md

```diff
@@ -0,0 +1,253 @@
+# @loopback/extension-metrics
+
+This module contains a component that reports metrics of Node.js, LoopBack
+framework, and your application to [Prometheus](https://prometheus.io/).
+
+## Stability - Experimental
```

Comment:
It makes we wonder how to better emphasize the experimental status. How about using all-caps and adding an emoji, e.g. 🛑 🚨 or ⚠️?

```suggestion
## Stability: ⚠️ EXPERIMENTAL ⚠️ 
```